src/platforms/javascript/guides/ember/ember2.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Ember 2.x
+redirect_from:
+  - /clients/javascript/integrations/ember/
+---
+
+To use Sentry with your Ember application, you will need to use Sentry’s browser JavaScript SDK: `@sentry/browser`.
+
+On its own, `@sentry/browser` will report any uncaught exceptions triggered from your application.
+In order to use ESM imports without any additional configuration, you can use `ember-auto-import`
+by installing it with `ember install ember-auto-import`.
+
+Starting with version `5.x` our `Ember` integration lives in it's own package `@sentry/integrations`.
+You can install it with `npm` / `yarn` like:
+
+```bash
+# Using yarn
+yarn add @sentry/integrations
+
+# Using npm
+npm install @sentry/integrations
+```
+
+Then add this to your `app.js`:
+
+```javascript
+import * as Sentry from "@sentry/browser";
+import { Ember as EmberIntegration } from "@sentry/integrations";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [new EmberIntegration()],
+});
+```
+
+In case you are using the CDN version or the Loader, we provide a standalone file for every integration, you can use it
+like this:
```

Comment:
go ahead and remove this line as it's now part of the prior one.

Reply:
Fixed 👍 . I believe you can do multi-line suggestions now, iirc, https://github.blog/changelog/2020-02-26-multi-line-code-suggestions-beta/